<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibration_test1Class</class>
 <widget class="QWidget" name="calibration_test1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calibration_test1</string>
  </property>
  <widget class="QPushButton" name="btn_threshold">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>440</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定所选范围</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>240</y>
     <width>201</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>校准参数</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>u</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ulabel">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>v</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="vlabel">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>RFI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RFIlabel">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>RFD</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RFDlabel">
        <property name="text">
         <string>0.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>845</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="threshSlider">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>130</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>调整阈值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>160</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>65535</string>
   </property>
  </widget>
  <widget class="QLabel" name="threshValue">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>100</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="calibration_test1.qrc"/>
 </resources>
 <connections/>
</ui>
